src/seedu/addressbook/ui/TextUi.java

```diff
@@ -164,5 +165,16 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    private String getPrintableString(Printable... printables) {
+    	final StringBuilder formatted = new StringBuilder();
+    	for(Printable i : printables) {
+    		formatted.append(i.getPrintableString()).append("\n");
```

Comment:
Be aware that there are a few ways to concatenate strings in Java, such as `String.join`, etc.